docker/e2e-docker-test.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
```

Comment:
does this also run on windows? Should we not use the jest setup we already have for running these tests?